--- argp.old/argp-help.c	2008-06-28 06:35:14.000000000 +0000
+++ argp/argp-help.c	2008-06-28 06:37:10.000000000 +0000
@@ -49,6 +49,9 @@ char *alloca ();
 #include <ctype.h>
 /* Does any system still need malloc.h? If so, we'd need a configure
    test. */
+#ifdef _WIN32
+#include <malloc.h>
+#endif
 
 #ifndef _
 /* This is for other GNU distributions with internationalized messages.  */
--- argp.old/argp-parse.c	2008-06-28 06:36:46.000000000 +0000
+++ argp/argp-parse.c	2008-06-28 06:37:05.000000000 +0000
@@ -46,6 +46,9 @@ char *alloca ();
 #include <unistd.h>
 #include <limits.h>
 #include <assert.h>
+#ifdef _WIN32
+#include <malloc.h>
+#endif
 
 #ifndef _
 /* This is for other GNU distributions with internationalized messages.
